src/instances.ts

```diff
@@ -266,17 +272,25 @@ function successfulTypeScriptInstance(
       .getProgram()
       .getProgram();
   } else {
-    const servicesHost = makeServicesHost(scriptRegex, log, loader, instance);
+    const cachedServicesHost = makeServicesHost(
```

Comment:
Could this be `servicesHost` instead please? It may or may not be cached depending upon the flag.

Reply:
Yeah, no problem.

Do I need to rename `serivcesHost` property in `CachedServicesHost` interface too? Also, it is possible, do I need to rename `CachedServicesHost` interface? Change it to `type` instead of `interface`?

Reply:
> Do I need to rename serivcesHost property in CachedServicesHost interface too? 

No I think it's fine.